api/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/api/RedisPoller.kt

```diff
@@ -8,7 +8,8 @@ import no.nav.helsearbeidsgiver.utils.log.sikkerLogger
 import java.util.UUID
 
 private const val MAX_RETRIES = 10
-private const val WAIT_MILLIS = 500L
+private const val WAIT_MILLIS_DEFAULT = 500L
+private val WAIT_MILLIS = List(MAX_RETRIES) { 100 + (it * it * 50L) } // Gir [100, 150, 300, 550, 900, 1350, 1900, 2550, 3300, 4150]
```

Comment:
Liker tanken, men synes det hadde vært bedre med `List(MAX_RETRIES) { 100L * (1 + it) }`.